wallet/src/main/scala/org/bitcoins/wallet/api/WalletApi.scala

```diff
@@ -37,6 +37,7 @@ sealed trait WalletApi {
 
   val nodeApi: NodeApi
   val chainQueryApi: ChainQueryApi
+  val creationTime: Long
```

Comment:
This should really be typed as `java.time.Instant` -- avoid using `Long` to represent a time at all costs as it is _extremely_ easy to get confused with handling time.